### ISSUE TITLE
Chapter 3. Page 98. Error when using textbook code

### ISSUE THREAD

**Issue by julianpraxis** [ageron/handson-ml#30]
Hi.
I get this.  
![image](https://cloud.githubusercontent.com/assets/17202910/26770523/4221844a-497d-11e7-99a6-1b74ae0e412d.png)

**Comment by ageron** [comment]
Hi Julian,

The error message says that the `plot_digits()` method is not defined, but it is defined in cell [8]. The notebooks are meant to be run in order, starting from the top. I'm guessing you skipped cell [8]. If so, the solution is to restart the kernel and run all cells up to the cell you are looking at.  Or perhaps you deleted cell [8] by mistake? In this case, you want to restore the original version of the notebook?

Hope this helps.

**Comment by julianpraxis** [comment]
@ageron I was following the book, and plot_digits() is not defined in the book.  I relied instead on your notebook's cells [6] and [7].  Since cell [8] has a comment that it is "extra", I skipped cells [8] and [9] in your notebook.  By the time I got to cell [72] in your notebook, I had forgotten about cell [8].

So, it was sort of an oversight in my part.  However, I would suggest that a comment might be added to cell [72] reminding the reader to ensure that cell [8] is used because it is where plot_digits() is defined.

Thanks for the prompt response.
